### DISCUSSION TITLE
Search Pane customization

### DISCUSSION THREAD

**Discussion by fredczj** [thegeeklab/hugo-geekdoc#517] [Q&A] [answered]
Hi

What is the best way to customize the `css` (background most of) of the **search result pane** ?

I feel that there is not enough differences between the body background and this search pane background so that the search results confuses with navigation panel.

Thanks in advance

**Comment by xoxys** [comment, marked as answer]
Hi, Please use discussions instead of issues for general questions. How customization works is described in the [docs](https://geekdocs.de/features/theming/). To customize the search result list, you have to adjust the class `.gdoc-search__list` in your `custom.css` file.

**Comment by fredczj** [comment]
Fine, thanks.

Indeed, I added this to `custom.css`:

``` css
.gdoc-search__list {
    background: var(--search-background);
}
```

and defining `--search-background: #123;` per color-theme.
